cloud-data-protection/src/stores/Store.tsx

```diff
@@ -0,0 +1,10 @@
+import {configureStore} from "@reduxjs/toolkit";
+import userReducer from '../features/userSlice';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
import userReducer from "../features/userSlice";
```